node-engine/reference/cli/wpe/apps/create.mdx

```diff
@@ -75,6 +79,41 @@ Providing custom `domains` is optional. If you do want to connect a custom domai
 
 **NOTE: We currently only support one custom domain.**
 
+## Create From a zip file
+
+**Note:** The zip app flow only works with wpe CLI version 1.0.0 or greater.
+
+You can create an Atlas app from a `json` file configured for zip uploads. All requirements from the standard `wpe create` flow apply, with a few exceptions.
+
+To make your `json` file zip configurable, set the `"repo"` parameter to `"uploadable"`. Then, add a parameter named `"zip_file_path"` with the directory path to your zip file in your environment variables. 
+
+Finally, ensure that you include an environment `id` to prevent creating duplicate environments.
+
+```json
+{
+  "name":"your_app_name",
+  "repo": "uploadable",
+  "region":"EU-W",
+  "environments": [
+    {
+      "domains":[],
+      "id":"hdcejft13e2rcn345",
+      "name": "someenvname",
+      "zip_file_path": "path/to/file.zip",
+      "branch": "-",
```

Comment:
I know it's really stupid, but adding "branch":"-" is also a necessary component of the config file, so it's worth to highlight it imo